stubs/setuptools/METADATA.toml

```
@@ -1,4 +1,4 @@
-version = "71.1.*"
+version = "72.1.*"
```

Setuptools 72.2 has been released as well. The only change is a merge of their distutils update to support more PathLike.

Whilst this means setuptools stubs *could* be updated to support more `StrPath`, that's a task I should do in setuptools as well following https://github.com/pypa/setuptools/pull/4504 . So I'll keep both in sync then.

In other words, this PR can bump to 72.2 w/o further changes
```suggestion
version = "72.2.*"
```